### PR TITLE
change wording Admin UI introduction (SCRD-7739)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

**Comment by jeremy-moffitt** [comment]
> @csymons-suse In think it should be other way.
> http://Deployer Management IP:9085 is used to access SUSE OpenStack Cloud Installer for Installing Cloud from UI.
> https://Deployer VIP:9085 is used to access CLM Admin Console for performing Cloud Operations after Cloud Installation is done.

Is the management network setup before the cloud has deployed? I thought it wasnt setup yet... you can tweak the settings for the management network in Day 0 cant you?

**Comment by mserigithub** [comment]
> > @csymons-suse In think it should be other way.
> > http://Deployer Management IP:9085 is used to access SUSE OpenStack Cloud Installer for Installing Cloud from UI.
> > https://Deployer VIP:9085 is used to access CLM Admin Console for performing Cloud Operations after Cloud Installation is done.
> 
> Is the management network setup before the cloud has deployed? I thought it wasnt setup yet... you can tweak the settings for the management network in Day 0 cant you?

@jeremy-moffitt Yes we can tweak the settings of management network during install. We can also assign the ip to the deployer from management network range during sles install too.

**Comment by jeremy-moffitt** [comment]
@mserigithub thanks for clarifying... @csymons-suse  please go ahead with Mohans suggestion, thanks!

**Comment by csymons-suse** [comment]
here's a screenshot of the latest version.
adminui_1

File: xml/day2_ui_documentation.xml [hunk 1]
```diff
@@ -9,26 +9,31 @@
  <title>&lcm; Admin UI User Guide</title>
  <para>
   The &lcm; Admin UI is a web-based GUI for viewing and managing the
-  configuration of an installed cloud.
+  configuration of an installed cloud. After successfully deploying the cloud,
+  the last screen will display a link to the CLM Admin UI. (For example, see
```
Comment (dmpop) [inline xml/day2_ui_documentation.xml]: Our [Style Guide](http://docserv.nue.suse.com/documents/Style_Guide/docu_styleguide/single-html/#sec.tense) recommends using present tense.

File: xml/day2_ui_documentation.xml [hunk 2]
```diff
@@ -9,26 +9,31 @@
...
-  will typically be <command>https://<replaceable>DEPLOYER_VIP</replaceable>:9085</command>
-  but may vary with the cloud configuration.</para>
 <section xml:id="_accessing_clm_admin_ui">
  <title>Accessing the Admin UI</title>
  <para>
-  Open a web browser to the URL that hosted the &lcm; &install_ui;, typically
-  <command>https://<replaceable>DEPLOYER_MGMT_NET_IP</replaceable>:9085</command>,
-  though the port may be different depending on the version and
-  configuration.
+  Open a web browser to
```
Comment (dmpop) [inline xml/day2_ui_documentation.xml]: Honest question: can you actually open a browser to? Perhaps just _Go to_?

File: xml/day2_ui_documentation.xml [hunk 3]
```diff
@@ -9,26 +9,31 @@
...
+  The
+  <literal><replaceable>DEPLOYER_MGMT_NET_IP</replaceable>:<replaceable>PORT_NUMBER</replaceable></literal>
+  is not necessarily the same for all installations, and can be displayed with
+  the following command:
  </para>
+ <screen>&prompt.ardana;openstack endpoint list --service ardana --interface admin -c URL</screen>
  <para>
   Accessing the &lcm; Admin UI requires access to the <emphasis
   role="bold">MANAGEMENT</emphasis> network that was configured when the Cloud
   was deployed. Access to this network is necessary to be able to browse to the
```
Comment (dmpop) [inline xml/day2_ui_documentation.xml]: _browse to_ -> _access_

**Comment by csymons-suse** [comment]
Made suggested changes. Thank you.

**Pull request merged**